### PR TITLE
A few requested enhancements to the Ansible guide

### PR DESCRIPTION
**Pull request by rpelisse**
See https://github.com/quarkusio/quarkus/issues/33256

### CODE REVIEW

File: docs/src/main/asciidoc/ansible.adoc [hunk 1]
```diff
@@ -22,6 +22,8 @@ You'll need to https://docs.ansible.com/ansible/latest/installation_guide/intro_
 $ ansible-galaxy collection install middleware_automation.quarkus
 ----
 
+Note: the Ansible collection we just installed only supports RHEL, Fedora and other Linux distribution using RPMs. Ansible defines these as "RedHat family". Using the content on other platform (Windows, Debian, Ubuntu,...), while not impossible, will require a few adjustements.
```
Comment (gsmet) [inline docs/src/main/asciidoc/ansible.adoc:25]:
```suggestion
NOTE: The Ansible collection we just installed only supports RHEL, Fedora, and other Linux distribution using RPMs. Ansible defines these as "Red Hat family". Using the content on other platform (Windows, Debian, Ubuntu, ...), while not impossible, will require a few adjustments.
```

File: docs/src/main/asciidoc/ansible.adoc [hunk 2]
```diff
@@ -152,6 +154,13 @@ Here again, it checks that the appropriate OpenJDK is available on the target sy
 
 With those requirements in place, the jar can be deployed on the target, along with the required configuration for the app integration into systemd as a service. Any change to the systemd configuration requires reloading its daemon, which the collection ensures will happen whenever it is needed. With all of that in place, the only thing that remains is to start the service itself, which Ansible will also take care of.
 
+Note: by default the Ansible collection for Quarkus will install and use the OpenJKD 17 available in the Yum repositories of the target host (or the controller). If you wish to use a different version of the OpenJDK, you need to define the following variable:
```
Comment (gsmet) [inline docs/src/main/asciidoc/ansible.adoc:157]:
```suggestion
NOTE: By default, the Ansible collection for Quarkus will install and use the OpenJDK 17 available in the Yum repositories of the target host (or the controller). If you want to use a different version of OpenJDK, define the following variable:
```

**Review by gsmet — CHANGES_REQUESTED** [review]
Thanks. I spotted a few issues.

Note that using either:
```
NOTE: Your note.
```

or
```
[NOTE]
====
Your note.
====
```
will make the note an Asciidoc note with proper formatting.

**Comment by rpelisse** [comment]
@gsmet thanks! I did not realized there was a syntax for that. I also fixed the few typos you mentioned, expect for RedHat family, because that the value used by Ansible:

```
"ansible_os_family": "RedHat",
```

**Review by gsmet — APPROVED** [review]
I pushed an additional commit with some additional tweaks.

**Pull request merged**